test/e2e/specs/hello.test.js

```diff
@@ -12,10 +12,15 @@ describe( 'hello', () => {
 
 	it( 'Should show the New Post Page in Gutenberg', async () => {
 		expect( page.url() ).toEqual( expect.stringContaining( 'post-new.php' ) );
+		// Should display the title.
 		const title = await page.$( '[placeholder="Add title"]' );
 		expect( title ).not.toBeNull();
+		// Should display the Preview button.
 		const postPreviewButton = await page.$( '.editor-post-preview.components-button' );
 		expect( postPreviewButton ).not.toBeNull();
+		// Should display the Post Formats UI.
+		const postFormatsUi = await page.$( '.editor-post-format' );
+		expect( postFormatsUi ).not.toBeNull();
```

Comment:
Thanks for the e2e test, I wonder if this test should be separate for clarity :) 

Actually, even testing the preview button seems odd in this "hello world" test but I guess it's fine for now.

Reply:
At some point I'd like to flesh out the e2e tests and make sure we have tests for every element.

Reply:
I personally don't think we should have e2e tests for everything :). e2e have maintenability and debugging costs that sometimes I prefer more granular unit tests while keeping the e2e tests for the main regressions we want to prevent from.